src/prepare.js

```diff
@@ -1,16 +1,31 @@
 import { readFile, writeFile } from "fs/promises";
 import { getPluginFiles } from "./util.js";
+import isPrerelease from "./is-prerelease.js";
 
 export async function prepare(_, context) {
     const version = context.nextRelease.version;
     const fileMap = await loadFileMap();
     const { minAppVersion } = fileMap.get("manifest.json");
 
+    // TODO(TfTHacker/obsidian42-brat#93): Remove if BRAT updates to GitHub release based workings
+    // Pre-release, add `manifest-beta.json` key to filemap
+    if (isPrerelease(context.branch)) {
+        fileMap.set("manifest-beta.json", {
+            ...fileMap.get("manifest.json"),
+        });
```

Comment:
I'm not too familiar with BRAT, but I have seen `manifest-beta.json` files laying around in plugin repos.

Would it make sense, from your desired workflow's point-of-view, to load the pre-existing `manifest-beta.json` if it exists, or is replacing it entirely w/ the release `manifest.json` always OK?

Reply:
To my understanding, the `manifest-beta.json` is only used to indicate the beta version to BRAT. However, for the sake of consistency, I think it is best to use `manifest.json` as the source from which we create the `manifest-beta.json`.